ouroboros-network-api/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -38,6 +38,8 @@ data NodeToClientVersion
     -- ^ added @GetPoolDistr, @GetPoolState, @GetSnapshots
     | NodeToClientV_15
     -- ^ enabled @CardanoNodeToClientVersion11@, i.e., Conway
+    | NodeToClientV_16
```

Comment:
@amesgen I just noticed, thanks to you, that currently released version is NodeToClinetV_15

https://github.com/input-output-hk/ouroboros-network/blob/e4a222f8e0f5d55591cc14b062332a8a61e85724/ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs#L531

Does this mean I should revert back addition of `NodeToClientV_16` and this becomes much smaller? All I need is addition of `ShelleyNodeToClientVersion7`, am I reading this right?

Reply:
Right; yes, you can include the changes in `V_15` as Conway is unreleased :+1:

Reply:
`NodeToClientV_16` removed.